concurrent/master.go

```diff
@@ -2,11 +2,11 @@ package concurrent
 
 import (
 	"context"
-	"errors"
 	"fmt"
-	"runtime/debug"
 	"sync"
 	"sync/atomic"
+
+	stderr "github.com/pkg/errors"
```

Comment:
Why name this `stderr`? If you've removed the stdlib `errors` package, should just be able to use `errors.MethodName` directly?

Reply:
unfortunately it clashes with the `errors` module in the gateway itself